### ISSUE TITLE
Card: asLink missing ds focus outline

### ISSUE THREAD

**Issue by oddvernes** [digdir/designsystem#4762]
It has the browser default outline currently

<img width="843" height="167" alt="Image" src="https://github.com/user-attachments/assets/f0757427-a5a4-4718-a9dd-ed11f0faf7ea" />
